src/mappings/Competition/mapping.ts

```diff
@@ -100,6 +140,31 @@ export function handleNewVote(event: NewVote): void {
       suggestion.save();
     }
     vote.save();
+
+    let competition = Competition.bind(event.address);
+    competitionProposal.winningSuggestions = [];
+    let winningSuggestions = competitionProposal.winningSuggestions;
+    debug(competitionProposal.suggestions.toString() + '');
```

Comment:
does this log is require ? 
please use 'logs' if there is a need to log something.

Reply:
No this was left there by mistake